.github/workflows/e2e.yml

```
@@ -147,6 +147,18 @@ jobs:
       DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
       DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
 
+  e2e-vcluster:
+    if: ${{ inputs.e2e_test_suites == 'all' || inputs.e2e_test_suites == 'hostpath' || inputs.e2e_test_suites == '' }}
```

We want to pick this test suite if vcluster was chosen from the drop down
```suggestion
    if: ${{ inputs.e2e_test_suites == 'all' || inputs.e2e_test_suites == 'vcluster' || inputs.e2e_test_suites == '' }}
```